src/main/kotlin/com/ampnet/auditornode/model/websocket/WebSocketMessages.kt

```diff
@@ -35,7 +35,7 @@ enum class InputType {
     BOOLEAN, NUMBER, STRING;
 
     companion object {
-        fun parse(value: String): InputType? = values().find { it.name == value.toUpperCase() }
+        fun parse(value: String): InputType? = values().find { it.name == value.uppercase() }
```

Comment:
I cannot find where did you define this uppercase() method? I cannot compile it.

Reply:
I did not define it - it is from Kotlin 1.5.0 standard library. If you cannot compile the project in IntelliJ, make sure you re-imported the project via Gradle. If that still gives you trouble, try deleting the project locally and do a fresh import.